README.md

```diff
@@ -360,8 +360,18 @@ $ gosec -fmt=json -out=results.json -stdout *.go
 $ gosec -fmt=json -out=results.json -stdout -verbose=text *.go
 ```
 
+
+
 **Note:** gosec generates the [generic issue import format](https://docs.sonarqube.org/latest/analysis/generic-issue/) for SonarQube, and a report has to be imported into SonarQube using `sonar.externalIssuesReportPaths=path/to/gosec-report.json`.
 
+Two output formats (json + any of the other valid formats) from one run can be achieved by providing the `-convert` flag which takes two arguments separated by a comma: \<format>,\<file>.
+```bash
+# Write output in json format to log.json as well as output in text format to log.txt
+$ gosec -fmt=json -out=log.json -convert=text,log.txt ./...
```

Comment:
I am not sure if this is the most descriptive way to do this. I would rather allow multiple formats something like:

```
gosec -fmt=json,text -out=log ./...
```
and attached the file extensions according with the format